services/gateway/api/types/run.go

```diff
@@ -76,9 +76,10 @@ type RunResponseTask struct {
 }
 
 type RunTaskResponse struct {
-	ID     string                `json:"id"`
-	Name   string                `json:"name"`
-	Status rstypes.RunTaskStatus `json:"status"`
+	ID        string                `json:"id"`
+	Name      string                `json:"name"`
+	Status    rstypes.RunTaskStatus `json:"status"`
+	PodImages []string              `json:"pod_images"`
```

Comment:
For consistency, since Pod is an internal executor concept and not exposed in any api I'll use a slice of `Container` struct where we provide the `Image` field (in future we could add more fields if needed).

As a note, the first container is the one that will execute the task steps, the other optional containers are "service" container (i.e. a database required for tests)